Content.Shared/Actions/BaseActionPrototype.cs

```diff
@@ -93,7 +94,7 @@ public virtual void LoadFrom(YamlMappingNode mapping)
             serializer.DataReadFunction("description", string.Empty,
                 s => Description = FormattedMessage.FromMarkup(s));
 
-            serializer.DataField(this, x => x.Requires,"requires", null);
+            serializer.DataField(this, x => x.Requires,"requires", null!);
```

Comment:
null! is suss and this should be handled.

Reply:
Replaced with `string.Empty`.